internal/studio/studioHandlers.go

```diff
@@ -135,6 +135,8 @@ func (h *StudioHandlers) reRun(ctx context.Context, w http.ResponseWriter, r *ht
 		run.WithSkipGenerateLintReport(),
 		run.WithSkipSnapshot(true),
 		run.WithSkipChangeReport(true),
+		run.WithShouldCompile(false),
+		run.WithSkipTesting(true),
```

Comment:
Note in the studio, we don't care about testing / compilation. Although, now we have to worry about doing these things correctly and safely exiting the studio 🙃